src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -40,19 +40,56 @@
 import static org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection.FindBreakpointEvidenceSparkArgumentCollection;
 
 /**
- * Tool to run the sv pipeline up to and including variant discovery.
- * Expected input is a BAM with around 30x coverage.  Coverage much lower than that probably won't work well.
+ * Master tool to run the structural variation discovery pipeline.
```

Comment:
I believe I am the guy who put in the bad "master tool" description. 
Maybe we can change it "single tool"?